src/book.js

```diff
@@ -303,6 +306,8 @@ export class Book {
    *      generate preview content AND a downloadable zip
    */
   async createoutputfiles(isPreview) {
+    // set this to `true` to enable full-book previews (placeholder till it's in the UI)
+    const fullPreviewDevHack = true;
```

Comment:
Can you set this to false? We don't want it turned on for everyone, and it also looks like something in the logic chain makes previews break for both the output options that include aggregate files when it's on.